app/Assets/SizeVariantLegacyNamingStrategy.php

```diff
@@ -0,0 +1,106 @@
+<?php
+
+namespace App\Assets;
+
+use App\Contracts\SizeVariantNamingStrategy;
+use App\Facades\Helpers;
+use App\Models\Photo;
+use App\Models\SizeVariant;
+
+class SizeVariantLegacyNamingStrategy extends SizeVariantNamingStrategy
+{
+	protected string $originalExtension = '';
+
+	/**
+	 * Maps a size variant to the path prefix (directory) where the file for that size variant is stored.
+	 */
+	public const VARIANT_2_PATH_PREFIX = [
```

Comment:
See comment about App/Actions/Photos/Extensions/Constants.php :)

Reply:
Enumerations are only supported by PHP 8.1 and at the moment we still support PHP 8.0. We shouldn't change that now or uses will come to use with pitchforks and burning tar. :laughing: